### PR TITLE
1263. Class rename with Utils at the end plus suppress warnings where required

### PR DESCRIPTION
**Pull request by evasileva-epam**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-examples/src/main/java/io/github/epam/entities/UsersUtils.java  1
- jdi-performance/src/main/java/org/mytests/uiobjects/example/entities/DefaultsUtils.java  1
- jdi-light-html-tests/src/test/java/io/github/epam/test/data/MarvelHeroesUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4550041&bid=15306529)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/BaseValidations.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
+@SuppressWarnings("PMD.ClassNamingConventions")
 public class BaseValidations {
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/BaseValidations.java]: I think both BaseValidations should have "Utils" suffix
This one..

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/BaseValidations.java [hunk 2]
```diff
@@ -14,6 +14,7 @@
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
+@SuppressWarnings("PMD.ClassNamingConventions")
 public class BaseValidations {
```
Comment (RomE-HK) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/BaseValidations.java]: ..and that one

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/BaseValidationsUtils.java  1
- jdi-light-examples/src/main/java/io/github/epam/entities/UsersUtils.java  1
- jdi-light-html-tests/src/test/java/io/github/epam/test/data/MarvelHeroesUtils.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/BaseValidationsUtils.java  1
- jdi-performance/src/main/java/org/mytests/uiobjects/example/entities/DefaultsUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4550041&bid=15306529)

**Pull request merged**